pxr/imaging/plugin/hdRpr/rprApi.cpp

```diff
@@ -2070,6 +2070,11 @@ class HdRprApiImpl {
 #endif // RPR_EXR_EXPORT_ENABLED
             }
         }
+        if (preferences.IsDirty(HdRprConfig::DirtyRenderQuality) || force) {
+            int enableNormalization = preferences.GetCoreLegacyToon() ? 0 : 1;
+            RPR_ERROR_CHECK(m_rprContext->SetParameter(RPR_CONTEXT_NORMALIZE_LIGHT_INTENSITY_ENABLED, enableNormalization), "Failed to set toon mode");
```

Comment:
Please check.
Use Lagacy RPR Toon = true is RPR_CONTEXT_NORMALIZE_LIGHT_INTENSITY_ENABLED = 0
Use Lagacy RPR Toon = false is RPR_CONTEXT_NORMALIZE_LIGHT_INTENSITY_ENABLED = 1